src/generators/Minecraft.res

```diff
@@ -117,6 +118,25 @@ module Cuboid = {
       rotate: 0.0,
     }
 
+    // When a face is flipped both vertically and horizontally, this is the same as rotating 180 degrees.
```

Comment:
Nice work figuring this out 👍 

I suspect there is a case not covered when we apply `f === #None` and `flip !== #None`? E.g.

```
face
->Minecraft.Cuboid.Face.flip(#Vertical)
->Minecraft.Cuboid.Face.flip(#None)
```

You did a good job making the function succinct, but that succinctness also makes it a bit trickier to read and follow the logic. What do you think about expanding it out to something like this:

```
// When a face is flipped both vertically and horizontally, this is the same as rotating 180 degrees.
let flip = (face: t, flip: Generator_Texture.flip) => {
  let (newFlip, newRotate) = switch (face.flip, flip) {
  | (#None, #None) => (#None, face.rotate)
  | (#None, #Vertical) => (#Vertical, face.rotate)
  | (#None, #Horizontal) => (#Horizontal, face.rotate)
  | (#Vertical, #None) => (#Vertical, face.rotate)
  | (#Vertical, #Vertical) => (#None, face.rotate)
  | (#Vertical, #Horizontal) => (#None, face.rotate +. 180.0)
  | (#Horizontal, #None) => (#Horizontal, face.rotate)
  | (#Horizontal, #Vertical) => (#None, face.rotate +. 180.0)
  | (#Horizontal, #Horizontal) => (#None, face.rotate)
  }
  {
    rectangle: face.rectangle,
    flip: newFlip,
    rotate: newRotate,
  }
}
```

It's more verbose, but a lot easier to see what logic is being applied (this also helped me see the problem I mentioned above).